docs/book/SUMMARY.md

```diff
@@ -0,0 +1,18 @@
+# Summary
```

Comment:
Why is the README.md not good enough? This looks exactly same as that one.

Reply:
`summary.md` file is used to create side navigation bar, when html pages are generated using netlify. - https://5e8e250c0fc3db000638d75c--peaceful-haibt-c672b5.netlify.com/

content from the `README.md` will appear on the landing page under introduction. Currently I have put same content as side bar navigation in the bullet list, but basically we can put any content here.

Reply:
removed duplicate navigation from landing page.